test/functional/plugins_test.rb

```diff
@@ -90,54 +90,40 @@
   include FunctionalHelper
   let(:env) { { INSPEC_CONFIG_DIR: "#{config_dir_path}/input_plugin" } }
   let(:profile) { "#{profile_path}/inputs/plugin" }
+  def run_input_plugin_test_with_controls(controls)
+    cmd = "exec #{profile} --controls #{controls}"
+    run_result = run_inspec_process(cmd, json: true, env: env)
+    run_result.must_have_all_controls_passing
+    run_result.stderr.must_be_empty
+  end
```

Comment:
If you name it starting with `assert_`/`refute_`/`must_`/`wont_` then a failing test will report the location of the failure at the assertion call-site, rather than on the failure internal to that compound assert.

Reply:
I am not sure what you are referring to, nor what you are asking me to change. Could you explain using different words?